src/web/routes/application/steps/decision/decision.js

```diff
@@ -5,34 +5,24 @@ const { ELIGIBLE } = require('../common/constants')
 
 const toPounds = pence => (parseInt(pence, 10) / 100).toFixed(2)
 
-const getTitle = req => {
-  if (path(['session', 'claimUpdated'], req) === true) {
-    return req.t('decision.updatedClaimTitle')
-  }
-
-  return req.t('decision.newClaimTitle')
-}
-
 const getDecisionPage = (req, res) => {
   if (req.session.eligibilityStatus === ELIGIBLE) {
     const totalVoucherValueInPence = path(['session', 'voucherEntitlement', 'totalVoucherValueInPence'], req)
 
     return res.render('decision', {
-      title: getTitle(req),
-      subTitle: req.t('decision.subTitle',
-        {
-          totalVoucherValue: toPounds(totalVoucherValueInPence),
-          totalVoucherValueForFourWeeks: toPounds(totalVoucherValueInPence * 4)
-        })
+      title: req.t('decision.title'),
```

Comment:
how do decline decisions get rendered? Simple ignorance on my part...

Reply:
`decline decisions` .. do you mean unsuccessful application?